src/CMakeLists.txt

```
@@ -114,6 +114,7 @@ set(HEADERS
     V3Hasher.h
     V3HierBlock.h
     V3Inline.h
+    V3InlineCfuncs.h
```

Everywhere.  Funcs is a word so capitalized.

```suggestion
    V3InlineCFuncs.h
```